src/chains/celo/utils.ts

```diff
@@ -0,0 +1,75 @@
+//////////////////////////////////////////////////////////////////////////////
+// Utilities
+
+import type { Address } from 'abitype'
+import { trim } from '../../utils/data/trim.js'
+import type {
+  CeloTransactionRequest,
+  CeloTransactionSerializable,
+  TransactionSerializableCIP42,
+  TransactionSerializableCIP64,
+} from './types.js'
+
+export function isEmpty(
+  value: string | undefined | number | BigInt,
+): value is undefined {
```

Comment:
(note-to-self) Neat, learnt about "user-defined type guards" e.g. `value is undefined`